### DISCUSSION TITLE
Application Assets

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#5904] [General]
Is it currently possible to server application assets ? 
Like logos / images / custom icons and then pass the URL to the appropriate element or do we host these assets externally ? 
How are you guys currently dealing with this ? any ideas ? 

Being able to have Organization wide assets, and Application specific assets would be a neat feature i guess.

**Comment by melohagan** [comment]
Hey @poirazis 

There was an idea brought up before, and issue created from it: https://github.com/Budibase/budibase/discussions/5278

It would be nice to have!
